### ISSUE TITLE
Abandoned?

### ISSUE THREAD

**Issue by holderad** [s-s/dnscloak#11]
Has this project been abandoned after so much success on the iOS App Store?

DoH has finally moved into the spot light. Please don’t let this project die.

**Comment by thalamus** [comment]
Yeah, DNScrypt-proxy 2.0.28 was released today too, and it’s 2.0.25 in the TestFlight and 2.0.19 still on the App Store.
Please keep maintaining this, if only to stay current with upstream releases.

**Comment by radiochild577** [comment]
The current TestFlight build (2.2.1) is also set to expire in 14 days.

**Comment by thalamus** [comment]
I'm going to fork it and update it if it doesn't get done soon. I really need this app and there's absolutely no other alternative app.
I'm not really knowledgable at iOS development to actively develop it though, but I'll have a go - I have an active developer account so I could publish to TestFlight / App Store potentially, although the build process looks potentially complex and I guess there's dependency issues with some of the frameworks. 

If @s-s would give some tips on what things need updating between dnscrypt-proxy versions it would be appreciated.

**Comment by thalamus** [comment]
Yeah, so I spent a bit of time trying to get it to build without updating it (best to try the original source first prior to modifying it), and even using go v12 and dnscrypt-proxy v2.0.25 I couldn’t get it to build the framework that the dnscrypt binary sits inside.
It seems that there’s other non-standard setup required to get the build to work. I do plan to give it another shot at the weekend but I’m not overly confident that I’ll figure it out without help! 
I’m going to wipe out the whole lot, start again and see if I can work it out.

**Comment by rugabunda** [comment]
Dnscrypt 2.0.29 anonamous DNS features have basically depreciated all alternatives to Dnscrypt... I hope someome picks it up and carries the ball if not out good russie. Read more: https://www.reddit.com/r/dnscrypt/comments/dhoxah/anonymized_dns_is_here/

**Comment by michael1900** [comment]
Any good news? This has been abandoned?

**Comment by holderad** [comment]
Yes, it has been abandoned. I have moved myself, my family, and our company to NextDNS.

Check it out at https://nextdns.io

It's built and maintained by the director of engineering at Netflix and the former Head of Mobile & TV at Dailymotion. It is endorsed by the lead of Google's Jigsaw and the designer of the Twitter interface. 

I have to saw that I was skeptical and worried, but it is so impressive and works so fast, I have moved past my own cobbled together systems. When NextDNS does start charging, I'll be first in line to pay. The granular control and speed on mobile is fantastic. Not having to maintain, trouble shoot, and provide tech support for my own system: priceless.
